app/weeklyPicks/page.tsx

```diff
@@ -84,22 +81,26 @@ export default function WeeklyPickForm() {
         window.localStorage.setItem('team', data.type);
       }
 
-      const user = await account.get();
-      const response = await getUserWeeklyPick({
-        userId: user.$id,
-        weekNumber: '6622c75658b8df4c4612',
-      });
-
       const findTeamID = NFLTeams.find(
         (ele) => ele.teamName.toLowerCase() === teamSelect,
       );
 
-      response[user.$id] = { team: findTeamID?.$id, correct: true };
+      let appendNewResult;
+      if (userPick?.search(userId!) && userPick?.search(userId!) != -1) {
```

Comment:
I would stay way from post-fix TypeScript ! expression. It will present bugs in your code later and negates the reason TypeScript was used in the first place.

```suggestion
      if (userId && userPick?.search(userId) && userPick?.search(userId) != -1) {
```